setup.py

```diff
@@ -1,35 +1,53 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
 # Copyright (c) 2017 National Research Foundation (South African Radio Astronomy Observatory)
 # BSD license - see LICENSE for details
-from setuptools import setup, find_packages
-
+from setuptools import find_packages, setup
 
 setup(
-    name='nosekatreport',
-    description='Nose plugin for writing an annotated test report',
-    long_description=open('README.rst').read(),
-    author='cam',
-    author_email='cam@ska.ac.za',
-    license='BSD',
-    packages=find_packages(exclude=['ez_setup']),
-    install_requires=['Nose>=0.11.0', 'traceback2', 'ansicolors', 'numpy'],
-    url='',
+    name="nosekatreport",
+    description="Nose plugin for writing an annotated test report",
+    long_description=open("README.rst").read(),
```

Comment:
Adopt this,
See: https://github.com/ska-sa/katportalclient/blob/master/setup.py#L9+L23